tests/assets/git-repo-single-app-cluster-selector.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: fleet.cattle.io/v1alpha1
+kind: GitRepo
+metadata:
+  name: default-single-app-cluster-selector-9
+  namespace: fleet-default
+spec:
+  branch: master
+  paths:
+    - qa-test-apps/nginx-app
+  repo: https://github.com/rancher/fleet-test-data/
+  targets:
+    - clusterSelector:
+        matchLabels:
+          key_env: value_testing
```

Comment:
pls add exra line if you consider so

Reply:
Okay, I will add it.

Reply:
Unknowingly, I clicked on merge button. Sorry, I have changes made I will push to next immediate PR. :disappointed: